config/env/all.js

```diff
@@ -5,5 +5,5 @@ var keys = rootPath + '/keys.txt';
 module.exports = {
 	root: rootPath,
 	port: process.env.PORT || 3000,
-    db: process.env.MONGOHQ_URL
+    db: process.env.MONGOLAB_URL || 'localhost',
```

Comment:
This is very wrong and will not work. Where's the `mongodb://` prefix before `localhost`? Kindly look up how to construct a valid mongodb connection url